client/src/main/java/org/opensearch/ml/client/MachineLearningClient.java

```diff
@@ -361,7 +361,7 @@ default ActionFuture<MLUndeployModelsResponse> undeploy(String[] modelIds, @Null
      * Undeploy model
      * For additional info on deploy, refer: https://opensearch.org/docs/latest/ml-commons-plugin/api/model-apis/undeploy-model/
      * @param modelIds the model ids
-     * @param modelIds the node ids. May be null for all nodes.
+     * @param nodeIds the node ids. May be null for all nodes.
```

Comment:
"May be null for all nodes" - this sounds not a description but a suggestion. Why not just put "Use null for all nodes" if you are fully sure.

Reply:
I didn't change anything in the description. It was like that before. I just fixed `nodeIds`, before it was `modelIds`